packages/core-bridge/src/conversions.rs

```diff
@@ -161,84 +158,9 @@ impl ObjectHandleConversionsExt for Handle<'_, JsObject> {
             .expect("Core server gateway options must be valid"))
     }
 
-    fn as_telemetry_options(&self, cx: &mut FunctionContext) -> NeonResult<TelemetryOptions> {
```

Comment:
I didn't dig, but can you help me understand the removal of all the code herein?

Reply:
That's a hack so that he could progress on Worflow Update while I'm still wokring on metrics. Lang side wouldn't compile/run otherwise with latest core

Reply:
That obviously means that this PR can't be merged until metrics is completed.